Makefile

```diff
@@ -68,7 +68,7 @@ docker:
 
 # Run test
 test: tests/assets
-	$(PYTHON) -m pytest tests $(PYTEST_ARGS)
+	$(PYTHON) -m pytest  tests --durations=0 $(PYTEST_ARGS)
```

Comment:
And with this we get to see what difference in performance these settings make:
```
93.35s call     tests/test_recognize.py::test_recognize[pageparallel+metscache]
92.28s call     tests/test_recognize.py::test_recognize[pageparallel]
76.19s call     tests/test_recognize.py::test_recognize[]
74.83s call     tests/test_recognize.py::test_recognize[metscache]
55.92s call     tests/test_segment.py::test_run_blla[metscache]
55.11s call     tests/test_segment.py::test_run_blla[]
48.43s call     tests/test_segment.py::test_run_blla[pageparallel+metscache]
41.80s call     tests/test_segment.py::test_run_blla[pageparallel]
```

(In this case, it was only 2 pages – the scaling factor is not so great.)